### PR TITLE
fix alert message heading level

### PR DESCRIPTION
**Pull request by jenniemc**
## Description
The accessibility scan finds invalid heading order where the heading leap frog from h1 to h3. Update the alert message heading to be h2.

## Testing done
local and unit test

## Screenshots
**Single supporting facility but no available slot message**
image

**Single supporting facility message**
image

## Acceptance criteria
- [ ] Change the alert message heading to h3 for Single supporting facility but no slot scheduling message
- [ ] Change the alert message heading to h3 for Single supporting facility message

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/tests/project-cheetah/components/VAFacilityPage/index.unit.spec.jsx [hunk 1]
```diff
@@ -623,7 +629,13 @@ describe('VAOS vaccine flow: <VAFacilityPage>', () => {
...
     });
 
-    await screen.findByText(/We found one VA location for you/i);
+    // it should verify alert heading
+    expect(
+      await screen.findByRole('heading', {
+        level: 2,
+        name: 'We found one VA location for you',
+      }),
+    );
```
Comment (jbalboni) [inline src/applications/vaos/tests/project-cheetah/components/VAFacilityPage/index.unit.spec.jsx:638]: Generally, we should put `.to.be.ok;` or `to.exist;` at the end of these `expect()` statements. The `expect` statements aren't technically necessary here at all (if `findByRole` fails it'll thrown an error), but if we're going to use them we should fully write out what we're asserting.

File: src/applications/vaos/tests/project-cheetah/components/VAFacilityPage/index.unit.spec.jsx [hunk 2]
```diff
@@ -656,7 +668,13 @@ describe('VAOS vaccine flow: <VAFacilityPage>', () => {
...
```
Comment (jbalboni) [inline src/applications/vaos/tests/project-cheetah/components/VAFacilityPage/index.unit.spec.jsx:677]: Same comment as above.

**Review by jbalboni — COMMENTED** [review]
One minor comment about the tests

**Review by vbahinwillit — APPROVED** [review]
Approved with changes Jeff suggested.

File: src/applications/vaos/tests/project-cheetah/components/VAFacilityPage/index.unit.spec.jsx (hunk 1 above)
Comment (jenniemc) [reply src/applications/vaos/tests/project-cheetah/components/VAFacilityPage/index.unit.spec.jsx]: Makes total sense. Adding the assertion.

File: src/applications/vaos/tests/project-cheetah/components/VAFacilityPage/index.unit.spec.jsx (hunk 2 above)
Comment (jenniemc) [reply src/applications/vaos/tests/project-cheetah/components/VAFacilityPage/index.unit.spec.jsx]: Done.

**Pull request merged**